doc/source/whatsnew/v3.0.0.rst

```diff
@@ -225,7 +225,7 @@ ExtensionArray
 
 Styler
 ^^^^^^
--
+- Enh in :meth:`Styler.set_tooltips` provides alternative method to storing tooltips by using title attribute of td elements.
```

Comment:
```suggestion
- :meth:`Styler.set_tooltips` provides alternative method to storing tooltips by using title attribute of td elements. (:issue:`56981`)
```

Also can you put this under `Other Enhancements`?

Reply:
Done